CHANGELOG.md

```diff
@@ -213,6 +213,8 @@
 - `[privval]` Retry accepting a connection ([\#2047](https://github.com/cometbft/cometbft/pull/2047))
 - `[state]` Fix rollback to a specific height
   ([\#2136](https://github.com/cometbft/cometbft/pull/2136))
+- `[state]` Fix txSearch performance issue
```

Comment:
the entry should be a separate file in `.changelog/improvements/2855-fix-txsearch-performance.md`.

see https://github.com/informalsystems/unclog

Reply:
My bad, first PR here, will get that in tomorrow morning!

Reply:
Done https://github.com/cometbft/cometbft/pull/2855/commits/140bb5c2adbc947a22ec08bd992a06e84dac57c3